packages/create-plugin/templates/common/.config/webpack/webpack.config.ts

```diff
@@ -187,7 +187,7 @@ const config = async (env: Env): Promise<Configuration> => {
       new ReplaceInFileWebpackPlugin([
         {
           dir: DIST_DIR,
-          files: ['plugin.json', 'README.md'],
+          test: [/plugin\.json$/, /README\.md$/],
```

Comment:
since this is now a regex you should match start of the string and end of the string

this will (maybe?) match things usch as `test-plugin.json` or `INTERNAL-README.md